src/components/Settings/index.tsx

```diff
@@ -1,316 +1,2 @@
-            <MenuModalContent>
-              <MenuModalInner>
-                <TransactionSettings
-                  rawSlippage={userSlippageTolerance}
-                  setRawSlippage={setUserslippageTolerance}
-                  rawPreferredGasPrice={userPreferredGasPrice}
-                  setRawPreferredGasPrice={setUserPreferredGasPrice}
-                  deadline={ttl}
-                  setDeadline={setTtl}
-                  multihop={multihop}
-                  onMultihopChange={toggleMultihop}
-                />
-                <Text fontWeight="400" fontSize="14px" lineHeight="17px" marginTop="12px" marginBottom="8px">
-                  Interface settings
-                </Text>
-                <RowBetween marginBottom="12px">
-                  <RowFixed>
-                    <TYPE.body color="text4" fontWeight={500} fontSize="12px" lineHeight="15px">
-                      Toggle expert mode
-                    </TYPE.body>
-                    <QuestionHelper text="Bypasses confirmation modals and allows high slippage trades. Use at your own risk." />
-                  </RowFixed>
-                  <Toggle
-                    id="toggle-expert-mode-button"
-                    isActive={expertMode}
-                    toggle={
-                      expertMode
-                        ? () => {
-                            toggleExpertMode()
-                            setShowConfirmation(false)
-                          }
-                        : () => {
-                            toggle()
-                            setShowConfirmation(true)
-                          }
-                    }
-                  />
-                </RowBetween>
-                <Divider />
-                <RowBetween width="100%" marginTop="12px" marginBottom="12px">
-                  <MenuItem href="https://dxdao.eth.link/" rel="noopener noreferrer" target="_blank">
-                    <Info size={14} />
-                    About
-                  </MenuItem>
-                  <MenuItem href={CODE_LINK}>
-                    <Code size={14} />
-                    Code
-                  </MenuItem>
-                  <MenuItem href="https://discord.com/invite/4QXEJQkvHH">
-                    <MessageCircle size={14} />
-                    Discord
-                  </MenuItem>
-                </RowBetween>
-                <RowBetween alignItems="center" marginBottom="8px">
-                  <SwaprVersionLogo />
-                  <Divider horizontal style={{ height: 48 }} />
-                  <TYPE.body fontWeight={700} fontSize="8px" letterSpacing="3px" color="white">
-                    A DXDAO PRODUCT
-                  </TYPE.body>
-                </RowBetween>
-              </MenuModalInner>
-            </MenuModalContent>
-          </MenuModalContentWrapper>
-        </MenuModal>
-      </StyledMenu>
-    </>
-  )
-}
+export * from './SimpleSettings'
+export * from './Settings'
```

Comment:
What are the best practices for exporting?

I updated because of this: https://github.com/levelkdev/swapr-dapp/pull/851#discussion_r843682405.

Reply:
Well to be consistent with rest of the code index.ts its usually a base for the thing named on the folders its in. If its a case like this we would make 2 new folders and then inside them index.ts for each separate component. But if you implement changes requested that won't be needed...